floats/floats.go

```diff
@@ -567,6 +568,19 @@ func Norm(s []float64, L float64) float64 {
 	return math.Pow(norm, 1/L)
 }
 
+// ParseWithNA converts the string s to a float64 in v.
```

Comment:
Recently I read an article about NaN and NA. I had to look up what NA means (but on the other hand I never processed any external data). Now I would side with ParseWithNA.